module/VuFind/tests/integration-tests/src/VuFindTest/View/Helper/Root/ResultFeedTest.php

```diff
@@ -102,7 +102,8 @@ protected function getMockTranslator()
                 . '</strong> results of <strong>%%total%%</strong>',
         ];
         $mock = $this->getMockBuilder(\Laminas\I18n\Translator\TranslatorInterface::class)
-            ->getMock();
+            ->addMethods(['getLocale'])
```

Comment:
Rather than implementing this change in multiple places, it looks like there's an opportunity to simply leverage the \VuFindTest\Feature\TranslatorTrait in this test and reduce some existing redundancy.

Reply:
Done